lib/sequencer/src/execution/block_context_provider.rs

```diff
@@ -49,6 +51,7 @@ pub struct BlockContextProvider<Mempool> {
     pubdata_price_override: Option<U256>,
     native_price_override: Option<U256>,
     pubdata_price_provider: watch::Receiver<Option<u128>>,
+    blob_fill_ratio_provider: watch::Receiver<Option<Ratio<u64>>>,
```

Comment:
not necesarrily about this PR, but more comments would be very helpful here. E.g. `pending_block_context_sender` - it'd be benefitial to metnion that it's used in json RPC to accurately estimate new transactions (so that up-to-data block info is used for inccoming tx estiamtion).

Same applies to `blob_fill_ratio_provider` - we could just say that it affects the per-tx price that we pay for L1 inclusion